godtools/Managers/LanguagesManager.swift

```diff
@@ -144,15 +159,16 @@ class LanguagesManager: GTDataManager {
         }
     }
     
-    func setSelectedLanguageId(_ id: String) {
+    func setSelectedLanguageId(_ language: Language) {
```

Comment:
Let's change the name of the method to `setSelectedLanguage` since we are passing in a `Language` instead of a language ID